feature-group-definitions/page-break.yml

```
@@ -0,0 +1,31 @@
+spec:
+  - https://drafts.csswg.org/css-multicol-1/#break-before-break-after-break-inside
```

This is really just a redirect to the CSS Fragmentation Module, which defines the actual properties. I suggest to replace with (or at least to add):

```suggestion
  - https://drafts.csswg.org/css-break/#breaking-controls
```